app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ErrorsController < ApplicationController
+  before_action :set_format
```

Comment:
this converts all formats that get to the errors controller (e.g. pdf) to html. i can't think of a situation where we wouldn't want this